backend/compact-connect/docs/onboarding/JURISDICTION_COMPACT_ONBOARDING.md

```diff
@@ -0,0 +1,170 @@
+# Jurisdiction Onboarding for Compact Connect
```

Comment:
# General Comments

* This looks great! Thanks for the thorough documentation- really helped me walk through and understand the codebase. I also appreciate your detailed commit messages, which answered some of my questions before I even walked through the code.
* I noticed we're copying a lot of the code for the lambda (e.g., the pagination and api_handler code and their corresponding tests). I don't see this as a huge issue if we only do so once or twice, but is there a plan to constrain the duplication going forward? Perhaps you don't expect to need to make similar lambdas again, or there are longer-term strategies for sharing code. No action needed on this PR, but I just wanted to throw it out there to make sure it's something we're thinking about.

Reply:
Great points, I discussed this briefly with @jusdino last week and we agree the duplicated utility, config, and testing code should be addressed if we add more directories like this. The way I've tackled this before is to add a common directory with the common modules inside, and deploy the common code as a lambda layer which is then shared across all the lambdas in the CDK app. There are of course other solutions we can consider but that is one of the simplest.

So yeah, we should definitely discuss during our next spring retro on how we would want to address common code for the project.

Reply:
Awesome. That makes a lot of sense, and we can cross that bridge when we get there.